content/200-orm/100-prisma-schema/20-data-model/50-database-mapping.mdx

```diff
@@ -111,9 +111,9 @@ Since most databases have a length limit for entity names, the names will be tri
 
 ### Using default constraint names
 
-When no explicit names are provided via `map` arguments Prisma will assume they follow the default naming convention.
+When no explicit names are provided via `map` arguments Prisma ORM will assume they follow the default naming convention.
 
-If you introspect a database the names for indexes and constraints will be added to your schema unless they follow Prisma's naming convention. If they do, the names are not rendered to keep the schema more readable. When you migrate such a schema Prisma will infer the default names and persist them in the database.
+If you introspect a database the names for indexes and constraints will be added to your schema unless they follow Prisma ORM's naming convention. If they do, the names are not rendered to keep the schema more readable. When you migrate such a schema Prisma will infer the default names and persist them in the database.
 
 #### Example
 
```

Comment:
Line 120 needs to be change to replace ` that will` with just `:` so that it introduces the code block.

(I can't do that via a suggestion, so you will need to commit that directly to the branch. If you prefer, I can also do that and commit to the branch - just let me know.)